src/main/java/attendance/io/view/InputView.java

```diff
@@ -0,0 +1,79 @@
+package attendance.io.view;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String INPUT_COMMAND_FORMAT =
+            "오늘은 %s입니다. 기능을 선택해 주세요.\n"
+                    + "1. 출석 확인\n"
+                    + "2. 출석 수정\n"
+                    + "3. 크루별 출석 기록 확인\n"
+                    + "4. 제적 위험자 확인\n"
+                    + "Q. 종료\n";
```

Comment:
java 15에서 추가된 [Text Blocks](https://www.baeldung.com/java-text-blocks) 기능을 활용해볼 수 있을 것 같아요. 

```suggestion
            """
            오늘은 %s입니다. 기능을 선택해 주세요.
            1. 출석 확인
            2. 출석 수정
            3. 크루별 출석 기록 확인
            4. 제적 위험자 확인
            Q. 종료
            """;
```

Reply:
감사합니다!

[refactor: InputView 입력 포맷 Text Blocks 활용](https://github.com/woowacourse/java-attendance/pull/126/commits/9dc8f13bca02d4562d2b313fcd09cabe46103149)